@ecomplus/storefront-components/src/js/ProductVariations.js

```diff
@@ -51,7 +51,16 @@ export default {
 
     orderedGrids () {
       return Object.keys(this.variationsGrids)
-    }
+    },
+
+    variationFromUrl () {
+      const urlParams = new URLSearchParams(window.location.search)
+      const variationId = urlParams.get('variationId')
+      if (variationId) {
+        return variationId
+      }
+      return false
```

Comment:
```suggestion
      if (typeof window === 'object') {
        const urlParams = new URLSearchParams(window.location.search)
        const variationId = urlParams.get('variation_id')
        if (variationId) {
          return variationId
        }
      }
      return null
```

1. melhor string | null, não mistura string com boleano
2. evita fatal em node env
3. url param em geral em snake_case

Reply:
Valeu